src/Notify/Resources/assets/src/controller.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import { Controller } from 'stimulus';
+
+/**
+ * @author Mathias Arlaud <mathias.arlaud@gmail.com>
+ */
+export default class extends Controller {
+    connect() {
+        const hubUrl = this.element.getAttribute('data-mercure-hub-url');
```

Comment:
Shouldn't you use [values](https://stimulus.hotwire.dev/reference/values) instead?

Reply:
Yes indeed! Thanks